testing/sdk_tasks.py

```diff
@@ -20,7 +22,7 @@ def fn():
                 running_task_names.append(t['name'])
             else:
                 other_tasks.append('{}={}'.format(t['name'], t['state']))
-        sdk_utils.out('Waiting for {} running tasks, got {} running/{} total:\n- running: {}\n- other: {}'.format(
+        log.info('Waiting for {} running tasks, got {} running/{} total:\n- running: {}\n- other: {}'.format(
```

Comment:
I this it's better to control higher level logging such as on this line and turn off lower level logging such as on a few lines below where we specify that `noisy=True` in `shakedown.wait_for`. Knowing that we're "Waiting for <something specific>" is far less noisier than generic spins so this allows greater flexibility for providing more accurate logging info based on the context.